src/Bicep.Cli/Commands/GenerateParametersFileCommand.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Cli.Arguments;
+using Bicep.Cli.Logging;
+using Bicep.Cli.Services;
+using Bicep.Core.FileSystem;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+
+namespace Bicep.Cli.Commands
+{
+    public class GenerateParametersFileCommand : ICommand
```

Comment:
Please make sure you add documentation for this new command to the CLI help text:

https://github.com/Azure/bicep/blob/8004f3a1ae0179018d273d8df73f9490e23e4db3/src/Bicep.Cli/Commands/RootCommand.cs#L66-L108